roles/pre_tasks/tasks/main.yml

```diff
@@ -1,10 +1,23 @@
 ---
 
 # Install absolutely necessary packages
+
+- name: Check if the absolutely necessary packages are installed
+  shell: "type {{ item }}"
+  with_items: "{{ tower_pre_tasks_packages }}"
+  register: type_packages
+  ignore_errors: yes
+
 - name: "Install absolutely necessary packages"
   yum:
-    name: "{{ tower_pre_tasks_packages }}"
-    state: present
+    name: "{{ package_item.item }}"
+    state: latest
```

Comment:
`[E403] Package installs should not use latest` -> replace with `present`

Reply:
Updated, Agree with changes